<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>661</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Electricity Consumption Prediction - Vietnam</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string> Forecast end time
[ format: mm/yyyy ]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_input">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>481</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="predict_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>530</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">&lt;property name=&quot;stylesheet&quot;&gt;
 &lt;string&gt;border: 50px solid black; border-radius: 5px;&lt;/string&gt;
&lt;/property&gt;
</string>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QWidget" name="plotWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>941</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">&lt;property name=&quot;stylesheet&quot;&gt;
 &lt;string&gt;border: 50px solid black; border-radius: 5px;&lt;/string&gt;
&lt;/property&gt;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>580</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">&lt;property name=&quot;stylesheet&quot;&gt;
 &lt;string&gt;border: 50px solid black; border-radius: 5px;&lt;/string&gt;
&lt;/property&gt;
</string>
    </property>
    <property name="text">
     <string>Export Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPast_data">
   <property name="text">
    <string>Past data</string>
   </property>
  </action>
  <action name="actionPredict_data">
   <property name="text">
    <string>Predicted data</string>
   </property>
  </action>
  <action name="actionPredicted_data_xlxs">
   <property name="text">
    <string>Predicted data (.xlxs)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
